index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="sv">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./lucky.style.css">
+    <link rel="Script" href="./lucky.script.js">
+    <script src="https://kit.fontawesome.com/be13722f69.js" crossorigin="anonymous"></script>
```

Comment:
Please use `<script src="https://kit.fontawesome.com/dd99507b81.js" crossorigin="anonymous"></script>` as it fixes the issue of some icons not showing (premium icons)